### ISSUE TITLE
Extend the WebExtension Transformer documentation

### ISSUE THREAD

**Issue by fregante** [parcel-bundler/website#834]
The docs are a little light: https://v2.parceljs.org/recipes/web-extension/

They should specify:
- what importing methods are supported (manifest keys, `_locales` folder, etc)
- [which aren't](https://github.com/parcel-bundler/parcel/issues/5758)
- which packages need to be installed; my go-to command is `npm i parcel@nightly @parcel/config-webextension@nightly @parcel/transformer-webextension@nightly`

cc @mischnic as the implementor of the transformer

Keywords: web extension webextensions webext web ext

**Comment by mischnic** [comment]
👍 

> my go-to command

You only need `parcel` and `@parcel/config-webextension`

>  the implementor of the transformer

Well, I reviewed the PR, but I have never developed a webextension myself

**Comment by fregante** [comment]
> > the implementor of the transformer
> 
> Well, I reviewed the PR, but I have never developed a webextension myself

Ahh I saw "merged by mischnic" in the header and wrongly assumed it was your PR instead of @101arrowz’s 😅

**Comment by 101arrowz** [comment]
Docs in general are a major issue for Parcel v2.

> what importing methods are supported (manifest keys, `_locales` folder, etc)

Every import method from `manifest.json` is supported. Methods from JavaScript are not. We can fix this by [adding RegEx's here](https://github.com/parcel-bundler/parcel/blob/v2/packages/transformers/js/src/JSTransformer.js#L44-L50) and [adding logic here](https://github.com/parcel-bundler/parcel/blob/v2/packages/transformers/js/src/visitors/dependencies.js#L40).

> which packages need to be installed

This is already documented. The sample code blocks just use `@parcel/config-webextension`, so most users would (presumably) just follow that.

**Comment by fregante** [comment]
> The sample code blocks just use `@parcel/config-webextension`, so most users would (presumably) just follow that.

Indeed, it seems to have changed since. However `@parcel/config-webextension` must be installed as `@next`

```
$ npm i parcel @parcel/config-webextension
$ parcel build manifest.json

🚨 Build failed.
@parcel/transformer-babel: config.setWatchGlob is not a function
```

```
$ npm i parcel @parcel/config-webextension@next
$ parcel build manifest.json

✨ Built in 1.18s
```

Being a _recipe_ it should probably include every step and/or a demo repository.
